sdk/core/core-tracing/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Breaking Changes
 
+- SpanOptions has been removed from OperationTracingOptions
```

Comment:
I want to call out that I purposely did not upgrade everyone at this time. side-by-side versions of core-tracing have been proven painful for various reasons. Even just having multiple versions in node_modules brought us pain (#16617). But locally I tested all packages against latest core-tracing bits, and this actually prepares us for backwards and forwards compatibility when we do release the next core-tracing version and bump everyone up.

Reply:
Why has this been removed? Context is appreciated!